sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/schema/query/JsonSchemaQueryProvider.java

```diff
@@ -0,0 +1,42 @@
+package io.stargate.sgv2.docsapi.service.schema.query;
+
+import io.stargate.bridge.proto.QueryOuterClass;
+import io.stargate.sgv2.common.cql.builder.QueryBuilder;
+import io.stargate.sgv2.docsapi.config.QueriesConfig;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class JsonSchemaQueryProvider {
```

Comment:
add a test for this class (see examples in #1840 and #1844).. this is also how you can confirm query is correct